modules/deploy/pages/deployment-option/cloud/aws-privatelink.adoc

```diff
@@ -31,9 +31,9 @@ TIP: Make sure to replace the variable values in the code examples on this page
 PUBLIC_API_ENDPOINT="https://api.cloud.redpanda.com"
 ----
 
-. In the https://cloud.redpanda.com/[Redpanda Cloud UI], go to **Namespaces** and select the namespace in which you want to create a cluster.
+. In the https://cloud.redpanda.com/[Redpanda Cloud UI], go to **Resource groups** and select the resource group in which you want to create a cluster.
 +
-Copy and store the namespace ID (UUID) from the URL in the browser.
+Copy and store the ID (UUID) from the URL in the browser.
```

Comment:
Wouldn't it be important to specify this is a resource group ID?

Reply:
Yes, thanks @malinskibeniamin! Also, can I chanage `NAMESPACE_ID`/`namespace_id` to `RESOURCE_GROUP_ID`/`resource_group_id`?

Reply:
I changed this to resource group ID. (Leaving `namespace_id` for now, per [this slack](https://redpandadata.slack.com/archives/C03P1GL0UF4/p1714662503962499?thread_ts=1714659193.614629&cid=C03P1GL0UF4).)